internal/ingress/metric/collectors/controller.go

```diff
@@ -127,6 +136,16 @@ func (cm *Controller) IncReloadErrorCount() {
 	cm.reloadOperationErrors.With(cm.constLabels).Inc()
 }
 
+// OnStartedLeading indicates the pod is not the current leader
```

Comment:
This is a bit confusing, `OnStartedLeading` should mean the pod is already the current leader, no?

Reply:
fixed